src/components/molecules/NavSearchBar/NavSearchBar.tsx

```diff
@@ -84,14 +88,26 @@ const NavSearchBar = () => {
     setSearchQuery("");
   }, []);
 
+  const clearSearchIcon = (
+    <img
+      className="nav__clear-search"
+      src="/icons/nav-dropdown-close.png"
+      alt="close button"
+      onClick={clearSearchQuery}
+    />
+  );
```

Comment:
I would write this as a sub-component that takes no props, rather than just a `const`, that way it can be used within the JSX as normal.